action.yml

```diff
@@ -99,6 +107,10 @@ runs:
         fi
 
         echo "pagesurl=$pagesurl" >> $GITHUB_ENV
+        
+        targetdir="$umbrella/pr-$pr"
+        echo "targetdir=$targetdir" >> $GITHUB_ENV
+        echo "pagespath=${targetdir/"$pagesbase"/}" >> $GITHUB_ENV
```

Comment:
If `$pagesbase` is e.g. `docs/` and `$targetdir` is e.g. `project-docs/pr-preview/pr-1`, this would result in `project-pr-preview/pr-1` which is not the intended result

I think it's reasonable to assume that situation won't happen, because if the user is specifying a base dir you'd expect their path to start with that base dir. But it would also be nice to permit formats like `docs`, `docs/`, `./docs/`, maybe even `/docs/`. I figure there must be some robust way to remove a path section in bash other than string substitution.

Reply:
I've updated it to make use of regex instead, will only replace if at the start of the string now.
Also made the end `/` optional to close off `pages-base-dir`.
```yml
umbrella-dir: ./docs/preview-docs
pages-base-dir: ./docs
# pagespath: preview-docs/pr-123

umbrella-dir: /docs/preview-docs
pages-base-dir: /docs
# pagespath: preview-docs/pr-123

umbrella-dir: docs/preview-docs
pages-base-dir: docs
# pagespath: preview-docs/pr-123

umbrella-dir: docs/preview-docs
pages-base-dir: docs/
# pagespath: preview-docs/pr-123
```

This does sort of depend on the user typing the beginning the same for `umbrella-dir` and `pages-base-dir` _(both start with `./path`, `/path` or `path`)_

But I think majority of the time this would happen naturally anyway, but added it to the documentation as well.